src/index.css

```
@@ -30,6 +31,12 @@
 
 @layer base {
   :root {
-    --radius: 0.5rem
+    --radius: 0.5rem;
   }
-}
\ No newline at end of file
+}
+
+@media (min-width: theme(--breakpoint-xs)) {
```

Biome (1.9.4)

[error] Unexpected unknown function: theme

Use a known function instead.
See MDN web docs for more details.

(lint/correctness/noUnknownFunction)